pkgs/applications/misc/qlandkartegt/garmindev.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, cmake, libusb }:
+
+stdenv.mkDerivation rec {
+  name = "garmindev-${version}";
+  version = "0.3.4";
+
+  src = fetchurl {
+    url = "https://bitbucket.org/maproom/qlandkarte-gt/downloads/${name}.tar.gz";
+    sha256 = "1mc7rxdn9790pgbvz02xzipxp2dp9h4hfq87xgawa18sp9jqzhw6";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ libusb ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    homepage = http://www.qlandkarte.org/;
+    description = "Garmin Device Drivers for QlandkarteGT";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ sikmir ];
+    platforms = with platforms; linux;
```

It fails to build on `aarch64`. I would recommend to set it to
```suggestion
    platforms = [ "x86_64-linux" ];
```

---

Fixed.